tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -430,6 +438,26 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(400));
 
+	printk("Update periodic advertising data 1...");
+	err = ll_adv_sync_ad_data_set(handle, AD_OP, sizeof(per_adv_data1),
+				      (void *)per_adv_data1);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(400));
+
+	printk("Update periodic advertising data 2...");
+	err = ll_adv_sync_ad_data_set(handle, AD_OP, sizeof(per_adv_data2),
+				      (void *)per_adv_data2);
```

Comment:
Should we test if the adv data is actually changed, by examining the data on the receiver? I guess this only tests whether the functions return true. Or is it too early to actually test that?

Reply:
It is verified in the PA report generation PR. But that PR depends on this PR, to set the PA data.